src/IntegrationUtils.ts

```diff
@@ -181,9 +201,52 @@ function getSelectedResource() {
 		}
 	});
 }
+		
+function getSelectedProperties() {
+	return new Promise <any> ( async (resolve, reject) => {
+		let hasMoreProperties: boolean = true;
+		let returnedProperties : string[] = [];
+		while (hasMoreProperties) {
+			let newProperty = undefined;
+			await vscode.window.showInputBox({
+				placeHolder: 'Specify the property name',
+				ignoreFocusOut: true,
+				validateInput: validateName
+			}).then ( async (propName) => {
+				await vscode.window.showInputBox({
+					placeHolder: 'Specify the property value',
+					ignoreFocusOut: true
+				}).then ( (propValue) => {
+					if (propValue) {
+						propValue = propValue.replace(/"/g, '\\"');
+						newProperty = `${propName}="${propValue}"`;
+					}
+				});
+			});
+			if (!newProperty) {
+				reject(new Error(`No Property provided`));
+				return;
+			} else {
+				const answer = await vscode.window.showQuickPick( ['No', 'Yes'], {
+					placeHolder: 'Are there more properties?',
+					canPickMany : false	});
+				if (!answer) {
+					reject(new Error(`No Property answer given`));
+					return;
```

Comment:
I am wondering if the code after the reject is even reached.

Reply:
See my earlier comment about reject/return

Reply:
Seems that works.

Reply:
Seems *what* works? reject? reject/return?

Reply:
executing code after the reject

Reply:
Cleaned up